src/ontology/mondo.Makefile

```diff
@@ -268,8 +269,42 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts (Aug 2023) pipeline
+# - https://github.com/monarch-initiative/mondo-ingest/issues/273
+.PHONY: address-medgen-conflicts-aug2023
+address-medgen-conflicts-aug202x3: address-medgen-conflicts-aug2023-deletes address-medgen-conflicts-aug2023-adds
+
+# - MedGen conflicts: dependencies
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+# - MedGen conflicts: analysis
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	-diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/medgen_conflicts_removals_diff_analysis.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+# - MedGen conflicts: deletes
+.PHONY: address-medgen-conflicts-aug2023-deletes
+address-medgen-conflicts-aug2023-deletes: mondo-edit.obo.tmp
+	mv mondo-edit.obo.tmp mondo-edit.obo
+
+mondo-edit.obo.tmp: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx mondo-edit.obo
+	python ../scripts/medgen_conflicts_removals.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -I mondo-edit.obo -o $@
+
+# - MedGen conflicts: adds
+.PHONY: address-medgen-conflicts-aug2023-adds
+address-medgen-conflicts-aug2023-adds: $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv
+
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/medgen_conflicts_add_xrefs.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
 
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
```

Comment:
## Next xref overlap analysis
You asked for another report that you called a "static analysis", where we report CUIs/CNs/UIDs that are `in_mondo`, `in_medgen`, or `both`. To do that, these removals/additions [need to be applied](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1306730253). Then you recommended 1 way to get my fresh input for that `mondo.sssom.tsv` was via [this `mondo-ingest` goal](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest.Makefile#L341).

I'll leave it to you as to whether you want to do this (a) ahead of [Next & final step: xref additions from MedGen mappings](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1306729478), or (b) after that step.

Reply:
@matentzn As long as everything looks good to you, this is ready to merge. Everything on my end [per the instructions from your comment in the issue](https://github.com/monarch-initiative/mondo-ingest/issues/273#issuecomment-1653431573) have been done. The "Next xref overlap analysis" above is the only remaining related immediate follow-up on my todo list.

Reply:
@joeflack4 I've now added `ROBOT_addMedGen_fromIngest.tsv`. This is the same file that is called `medgen-xrefs.robot.template.tsv` in the MedGen ingest.

Why do I rename it here? Just to be consistent with what's currently in the `templates/` dir:
```
ROBOT-partial-chromosome.tsv
ROBOT-partial-duplication.tsv
ROBOT_AML_NCIT_for_MONDO_2021-09-14.tsv
ROBOT_AddClinGenPref.tsv
ROBOT_CreateNewTerm.tsv
ROBOT_MergeTag.tsv
ROBOT_ObsoleteTag.tsv
ROBOT_addCongenitalSubclass-Mondo4069.tsv
ROBOT_addSubClassOf_Mondo3680.tsv

# And now also: 
ROBOT_addMedGen_fromConflictResolution.tsv
ROBOT_addMedGen_fromIngest.tsv
```